app/templates/components/learnergroup-overview.hbs

```diff
@@ -3,13 +3,19 @@
     {{learnerGroup.title}} {{t 'learnerGroups.members'}} ({{learnerGroup.usersOnlyAtThisLevel.length}})
   </div>
   <div class='detail-content'>
+    <div class="toggle-container">
+      {{toggle-onoff label="learnerGroups.multiEdit" on=multiEditModeOn action="toggleSwitch"}}
+    </div>
+
     {{learnergroup-members
       topLevelGroup=learnerGroup.topLevelGroup
       cohort=learnerGroup.cohort
       members=learnerGroup.usersOnlyAtThisLevel
       showMoveToCohortOption=true
       showMoveToTopLevelGroupOption=true
+      multiEditModeOn=multiEditModeOn
```

Comment:
I'm mostly OK with the way this breaks down using the learnergroup-members component with a multi-edit option, but it may be a lot cleaner to create a multiedit-learnergroup-members component and swap that out here instead of passing in this option.  Take a look, but feel free to leave it this way if that makes the most sense.